.travis.yml

```diff
@@ -1,61 +1,58 @@
-language: python
+language: generic
 sudo: false
-python:
-  - 2.7_with_system_site_packages
-  - 3.4
-  - 3.5
-env:
-  - ETS_TOOLKIT='qt4'
-  - ETS_TOOLKIT='wx'
-  - ETS_TOOLKIT='null'
-matrix:
-  exclude:
-    - python: 3.4
-      env: ETS_TOOLKIT='wx'
-    - python: 3.5
-      env: ETS_TOOLKIT='wx'
-    - python: 3.5
-      env: ETS_TOOLKIT='qt4'
-cache:
-  - pip
-  - ccache
+
 addons:
   apt:
     packages:
     - python-qt4
```

Comment:
since we use the edm packages we should not need to install 
python* packages and zlibg-dev, libpng-dev, libfreetype6-dev  from apt